apps/public-docsite-v9/src/Concepts/Migration/FromV0/Components/IconCatalog/ComparisonTile.styles.ts

```diff
@@ -1,14 +1,14 @@
-import { makeStyles, shorthands, tokens } from '@fluentui/react-components';
+import { makeStyles, tokens } from '@fluentui/react-components';
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=30765&commitId=ede0bd724d4663868b4c76f7e586ff38e35272f0&env=prod&clientType=FLUENTUI)


  <details><summary><b>Calendar Compat</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Calendar Compat.Custom icons.default.chromium.png|177| Changed|
|Calendar Compat.Default - RTL.default.chromium.png|177| Changed|
|Calendar Compat.Default.default.chromium.png|177| Changed|

</details>


  <details><summary><b>DatePicker Compat</b> 12 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|DatePicker Compat.default - Dark Mode.opened.chromium.png|163| Changed|
|DatePicker Compat.DateRange- month - Dark Mode.default.chromium.png|163| Changed|
|DatePicker Compat.DateRange- week.default.chromium.png|177| Changed|
|DatePicker Compat.DateRange- month.default.chromium.png|177| Changed|
|DatePicker Compat.default.opened.chromium.png|177| Changed|
|DatePicker Compat.DateRange- day.default.chromium.png|177| Changed|
|DatePicker Compat.DateRange- work week.default.chromium.png|177| Changed|
|DatePicker Compat.marked dates - Dark Mode.default.chromium.png|163| Changed|
|DatePicker Compat.showWeekNumbers.default.chromium.png|238| Changed|
|DatePicker Compat.marked dates.default.chromium.png|177| Changed|
|DatePicker Compat.showWeekNumbers - Dark Mode.default.chromium.png|202| Changed|
|DatePicker Compat.when rendering inline, it should not render behind relatively positioned elements.default.chromium.png|177| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>